File: 프로그래머스/1/12912. 두 정수 사이의 합/두 정수 사이의 합.py
def solution(a, b):
    if a == b:
        return a
    
    if a > b:
        # t = [i for i in range(a, b - 1, -1)]
        return sum([i for i in range(b, a+1)])
    else:
        # t = [i for i in range(a, b + 1)]
        return sum([i for i in range(a, b+1)])
    # return sum(t)

File: 프로그래머스/1/12912. 두 정수 사이의 합/두 정수 사이의 합.java
class Solution {
    public long solution(int a, int b) {
        if (a == b) return a;
        long answer = 0;

        if (a < b){
            int tmp = a;
            a = b;
            b = tmp;
        }
        for(int i = b; i <= a; i++){
            answer += i;
        }
        return answer;
    }
}